src/commands/custom/showmap.js

```diff
@@ -0,0 +1,28 @@
+const { SlashCommandBuilder, PermissionsBitField } = require("discord.js");
+const { loadReactionMap } = require("../../utils/reactionutils");
+
+module.exports = {
+    data: new SlashCommandBuilder()
+        .setName("showmap")
+        .setDescription("Display the current reaction map"),
+    async execute(interaction) {
+
+        // Check if the user has administrator permissions
+        if (
+            !interaction.member.permissions.has(
+                PermissionsBitField.Flags.Administrator
+            )
+        ) {
+            return interaction.reply({
+                content: "You do not have permission to use this command.",
+                ephemeral: true,
+            });
+        }
```

Comment:
This can use the defaultPermissionFlag or whatever it is (see /clear) to not have to worry about the permissions